Cargo.toml

```diff
@@ -11,28 +11,30 @@ repository = "https://github.com/slowtec/tokio-modbus"
 edition = "2018"
 
 [dependencies]
-bytes = "0.4"
+bytes = "0.5"
 byteorder = "1"
-futures = "0.1"
+futures = "0.3"
 log = "0.4"
-smallvec = { version = "0.6", default-features = false }
-tokio-io = "0.1"
-tokio-core = "0.1"
-tokio-codec = "0.1"
-tokio-proto = "0.1"
-tokio-service = "0.1"
+smallvec = { version = "1", default-features = false }
+tokio = { version = "0.2", features = ["tcp", "rt-core"] }
+tokio-util = { version = "0.2", features = ["codec"] }
+futures-util = "0.3"
 
 # Disable default-features to exclude unused dependency on libudev
-tokio-serial = { version = "3.3", optional = true, default-features = false }
+tokio-serial = { version = "4.3.3", optional = true, default-features = false }
+
+net2 = { version = "0.2", optional = true }
 
 [dev-dependencies]
 env_logger = "0.6"
+tokio = { version = "0.2", features = ["tcp", "rt-core", "macros"] }
```

Comment:
Only "tcp" and "macros" seem to be required?

Reply:
Confirm, fixed.